packages/utils/src/utils.ts

```
@@ -1,149 +1,128 @@
   res: Response | http.ServerResponse,
   logger: TwakeLogger,
-  callback: (obj: Record<string, string>) => void
+  callback: (obj: Record<string, string>) => void,
 ): void => {
-  let content = ''
-  let accept = true
+  let content = "";
+  let accept = true;
   const end = (): void => {
-    let obj
+    let obj;
```

Biome (2.4.9)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)